eel/__init__.py

```diff
@@ -306,9 +306,12 @@ def _process_message(message, ws):
         print('Invalid message received: ', message)
 
 
-def _get_real_path(path):
+def _get_real_path(path, frozen_with):
     if getattr(sys, 'frozen', False):
-        return os.path.join(sys._MEIPASS, path)
+        if frozen_with == 'cx_freeze':
+            return os.path.join(sys.executable, path)
+        elif frozen_with == 'PyInstaller':
+            return os.path.join(sys._MEIPASS, path)
```

Comment:
wouldn't it be easier to just do
```suggestion
        return os.path.join(getattr(sys, '_MEIPASS', sys.executable), path)
```
That way, user can seemlessly switch between the two